client/src/components/brushableHistogram/index.js

```diff
@@ -124,6 +124,63 @@ class HistogramBrush extends React.Component {
     };
   }
 
+  onBrushEnd(selection, x) {
+    return () => {
+      const { dispatch, field, isObs, isUserDefined, isDiffExp } = this.props;
+      const { brushX } = this.state;
+      const minAllowedBrushSize = 10;
+      const smallAmountToAvoidInfiniteLoop = 0.1;
+
+      if (d3.event.selection) {
+        let _range;
+
+        if (
+          d3.event.selection[1] - d3.event.selection[0] >
```

Comment:
question: do you need to abs() this before comparison (ie, can selection[1] be smaller than selection[0])?

Reply:
I believe d3 handles this internally, because swiping both left and right works.